src/ontology/fypo-odk.yaml

```diff
@@ -15,17 +15,33 @@ release_artefacts:
   - full
 primary_release: full
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
   products:
+    - id: go
+      make_base: TRUE
+      base_iris:
+        - http://purl.obolibrary.org/obo/GOCHE_
+        - http://purl.obolibrary.org/obo/GO_
+        - http://purl.obolibrary.org/obo/GOREL_
     - id: ro
+      use_base: TRUE
     - id: iao
+      use_base: TRUE
     - id: chebi
+      use_base: TRUE
```

Comment:
I think `use_base` is actually not needed here since we explicitly download the OBO-slim version with `mirror_from`.